process.c

```diff
@@ -4345,13 +4345,13 @@ exit_status_code(VALUE status)
 NORETURN(static VALUE rb_f_exit_bang(int argc, VALUE *argv, VALUE obj));
 /*
  *  call-seq:
- *     Process.exit!(status=false)
+ *    Process.exit!(status = false)
```

Comment:
Just like `exit`, this method also exists as `Kernel#exit!` should we add a call-seq for `exit!(status = false)` here to be consistent?

Reply:
Done.